source/features/swap-branches-on-compare.tsx

```diff
@@ -27,5 +27,8 @@ void features.add(__filebasename, {
 	include: [
 		pageDetect.isCompare
 	],
+	exclude: [
+		() => select.exists('.range-editor .octicon-arrow-both')
```

Comment:
I dont like this since if the selector changes we will never know that it broke.

@fregante ?

Reply:
I agree with that.
By the way, the init function uses the same way.
so I think this file assumes icon always exists.

Reply:
@yakov116 we use this pattern in a lot of features. Here the only reason not to use it is that DOM checks are not needed.